wandb/sdk/artifacts/artifact.py

```diff
@@ -191,6 +192,7 @@ def __init__(
         self._created_at: Optional[str] = None
         self._updated_at: Optional[str] = None
         self._final: bool = False
+        self._settings: Settings
```

Comment:
Does this work? I think the `Settings` object is a singleton, but it's still not instantiated here.